### PR TITLE
fix: increase image class size and compress logs (IN-1169)

### PR DESCRIPTION
**Pull request by edison-vflow**
### Fixes or implements 
* https://linear.app/voiceflow/issue/IN-1169/e2e-optimizations-increase-container-class-size-and-compress-logs-to

### Brief description. What is this change?

* Increase image class size 
* Compress logs and send to circleci artifacts 

### Checklist

- [x] Tested docker container class increase (4 CPUS and 8 Gig Ram is twice the previous size) 
image

- [x] Tested log compression 

image

### CODE REVIEW

**Review by xavidop — APPROVED** [review]

File: src/executors/node/smoke-executor.yml [hunk 1]
```diff
@@ -11,3 +11,4 @@ docker: # run the steps with Docker
       MKCERT_VERSION: v1.4.0
       NODE_VERSION: v20.10.0
       NODE_OPTIONS: --max_old_space_size=8192
+resource_class: large
```
Comment (xavidop) [inline src/executors/node/smoke-executor.yml:14]: lets do xlarge

**Review by xavidop — CHANGES_REQUESTED** [review]

File: src/jobs/e2e/collect-e2e-logs.yml [hunk 2]
```diff
@@ -112,7 +112,19 @@ steps:
...
          kubectl describe pod $component -n $DEV_ENV_NAME >> "${LOG_DIR:?}/${KUBE_STATE_DIR:?}/${last_component}-k8-state.log"
 
+  - run:
+      name: Compress logs
+      environment:
+        LOG_DIR: *log_dir
+      command: |
+        mkdir -p /tmp/logs/compressed
+        cp -r ${LOG_DIR:?} /tmp/logs/compressed
+        tar -czf /tmp/logs.tar.gz /tmp/logs/compressed
```
Comment (theProf) [inline src/jobs/e2e/collect-e2e-logs.yml:122]:
```suggestion
        cd /tmp/logs/compressed && tar -czf /tmp/logs.tar.gz .
```

File: src/executors/node/smoke-executor.yml (hunk 1 above)
Comment (edison-vflow) [reply src/executors/node/smoke-executor.yml:14]: Thought about it but was thinking we have been getting by with medium (2CPUs and 4Gig RAM) , so doubling the resources like we did now should be good.
Was also wondering how it looks costs wise.

Can do xlarge as well

**Review by xavidop — APPROVED** [review]

File: src/executors/node/smoke-executor.yml (hunk 1 above)
Comment (theProf) [reply src/executors/node/smoke-executor.yml]: will be good to jump it to get everyone working again, then can monitor to decrease if lots of buffer room

**Pull request merged**